test/test_data_handler.py

```diff
@@ -0,0 +1,60 @@
+'''
```

Comment:
This has to be modified with the fitter class as soon as available (to be kept in mind)

Reply:
yes indeed I agree, as soon as we have a first draft of the fitter we can replace this piece